src/Client/Dialogs/ReportDialog.razor.cs

```diff
@@ -54,7 +57,16 @@ protected override Task OnInitializedAsync()
     }
 
     protected async Task CopyToClipboard()
-        => await ClipboardService!.WriteTextAsync(ReportText ?? string.Empty);
+    {
+        if (Snackbar is null)
+        {
+            throw new ArgumentNullException(nameof(Snackbar));
```

Comment:
_"The exception that is thrown when a null reference (Nothing in Visual Basic) is passed to a method that does not accept it as a valid argument."_

So basically I am thinking if it's valid to throw ArgumentNullException when Snackbar is not an argument of a given method. 🤔 Personally I think InvalidOperationException is right choice. 

_The exception that is thrown when a method call is invalid for the object's current state._

Reply:
You are definitely right on this one, I've changed it within the latest commit. ☺